pkg/recipes/engine/engine_test.go

```diff
@@ -824,3 +973,85 @@ func Test_Engine_Execute_With_Secrets_Success(t *testing.T) {
 	require.NoError(t, err)
 	require.Equal(t, result, recipeResult)
 }
+
+func Test_Engine_Delete_With_Secrets_Success(t *testing.T) {
```

Comment:
nit: is this the naming convention we follow?

Reply:
the tests in this file are of similar format